public/react/SaveReports/SaveReports.tsx

```diff
@@ -2,14 +2,13 @@ import type { FC } from "react";
 import type { CanObservable } from "../hooks/useCanObservable";
 
 import React, { useEffect, useState } from "react";
-import DropdownMenu, { DropdownItem } from "@atlaskit/dropdown-menu";
 import { v4 as uuidv4 } from "uuid";
 import Button from "@atlaskit/button/new";
-import ChevronDown from "@atlaskit/icon/glyph/chevron-down";
 
 import { useAllReports, useCreateReport, useRecentReports } from "../services/reports";
 import SaveReportModal from "./components/SaveReportModal";
 import SavedReportDropdown from "./components/SavedReportDropdown";
+import ReportControls from "./components/SavedReportDropdown/ReportControls";
```

Comment:
Why is `ReportCntrols` going inside the SavedReportDropdown Modlet?

Reply:
fixed